cvat-ui/src/actions/auth-actions.ts

```diff
@@ -112,13 +113,20 @@ export const loginAsync = (username: string, password: string): ThunkAction => a
     }
 };
 
-export const logoutAsync = (): ThunkAction => async (dispatch) => {
+export const logoutAsync = (isHeaderAuthActive: boolean): ThunkAction => async (dispatch) => {
     dispatch(authActions.logout());
 
     try {
         await cvat.organizations.deactivate();
         await cvat.server.logout();
-        dispatch(authActions.logoutSuccess());
+
+        // Determine if user will be logged out through Datawiza Access Broker
+        // https://docs.datawiza.com/overview.html#what-is-datawiza-access-broker
+        if (isHeaderAuthActive) {
+            window.location.href = '/datawiza/ab-logout';
+        } else {
+            dispatch(authActions.logoutSuccess());
+        }
```

Comment:
We should dispatch authActions.logoutSuccess() for all types of authentication

Reply:
Fixed. The new implementation doesn't require a separate logout action.